packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -97,7 +112,7 @@ class TextMapPropagator {
       if (!trace.tags[key] || !key.startsWith('_dd.p.')) continue
       if (!this._validateTagKey(key) || !this._validateTagValue(trace.tags[key])) {
         log.error('Trace tags from span are invalid, skipping injection.')
-        return
+        continue
```

Comment:
The spec says to drop the entire header if any tag is invalid.

Reply:
The system tests seemed to be explicitly testing the opposite so might need to double check that. 🤔

Reply:
Fixed. Tests were changed since I had run it before.